pkg/scheduler/scheduler_test.go

```diff
@@ -201,6 +202,32 @@ func TestSchedulerCreation(t *testing.T) {
 	if err != nil {
 		t.Fatalf("Failed to create scheduler: %v", err)
 	}
+
+	// Test case for：when options.frameworkOutOfTreeRegistry's name already exists, create scheduler should fail.
```

Comment:
"Test case for when a plugin name in frameworkOutOfTreeRegistry already exist in defaultRegistry."

Reply:
> "Test case for when a plugin name in frameworkOutOfTreeRegistry already exist in defaultRegistry."

done, thanks.